### ISSUE TITLE
adapters now require node-fetch to be a dependency of the app

### ISSUE THREAD

**Issue by Rich-Harris** [sveltejs/kit#1076]
**Describe the bug**
Unanticipated side-effect of https://github.com/sveltejs/kit/pull/1066 — `node-fetch` isn't available when the app is built on e.g. Netlify:

![image](https://user-images.githubusercontent.com/1162160/115126106-ca22f280-9f9a-11eb-8d25-d6c326614dac.png)

We could make it a peer dependency of the adapters, but that would suck. I think it might be better if adapters took responsibility for bundling the output of `svelte-kit build` (with eg. `esbuild` — this functionality could also be exposed via the adapter utils). Then, if we exposed something like `@sveltejs/kit/install-fetch` for the relevant adapters to use (rather than expecting them to use the correct version of `node-fetch` and correctly assign the relevant bits to `globalThis`), it wouldn't mean needing `@sveltejs/kit` to be a production dependency.

**Severity**
Completely blocks use of the affected adapters, AFAICT

**Comment by chrisabruce** [comment]
I am hit by this as well trying to deploy to Netlify.  Is there any kind of work around?
```
{"errorType":"Error","errorMessage":"Cannot find package 'node-fetch' imported from /var/task/src/handler.mjs","trace":["Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'node-fetch' imported from /var/task/src/handler.mjs","    at packageResolve (internal/modules/esm/resolve.js:650:9)","    at moduleResolve (internal/modules/esm/resolve.js:691:18)","    at Loader.defaultResolve [as _resolve] (internal/modules/esm/resolve.js:784:11)","    at Loader.resolve (internal/modules/esm/loader.js:100:40)","    at Loader.getModuleJob (internal/modules/esm/loader.js:246:28)","    at ModuleWrap.<anonymous> (internal/modules/esm/module_job.js:47:40)","    at link (internal/modules/esm/module_job.js:46:36)"]}
```

**Comment by Rich-Harris** [comment]
1. downgrade
2. installing `node-fetch` as a prod dependency in your app might fix it?

**Comment by zommerberg** [comment]
Weirdly I have the same problem now, neither a downgrade or installing the node-fetch as prod dependency fixes it.

**Comment by zommerberg** [comment]
Fixed with netlify adapter 7

**Comment by mcurcija** [comment]
> Working fine after upgrading to `"@sveltejs/adapter-netlify": "^1.0.0-next.7"` and `"@sveltejs/kit": "1.0.0-next.80"`

I can confirm this as well. It also should be noticed that netlify-adapter now uses .netlify subdirectory so netlify.toml should be updated accordingly. The "node-fetch" dependency  should be removed completely as in  sveltejs/realworld.

**Comment by Rich-Harris** [comment]
This should be fixed now — #1091
